src/main/resources/templates/db_scripts/schema.json

```diff
@@ -398,13 +398,13 @@
         {
           "fieldName": "payment",
           "multiFieldNames": "amount, fromFundId, sourceInvoiceId, sourceInvoiceLineId, transactionType, expenseClassId",
-          "tOps": "ADD",
+          "tOps": "DELETE",
```

Comment:
for me better to delete competely index declaration, because during quick look this tOps can be missed and by mistake we can consider that index is exists.
But it is just my personal preference, we can leave as it, because this option is better for auditing purposes

Reply:
I see your point. My personal preference is to use DELETE, to make it explicit we want to delete it (and also avoid conflicts if we create a new index with the same name), and remove it after a few years to clean things up.